PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,46 +1,10 @@
-### Requirements
-
-* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
-* All new code requires tests to ensure against regressions
-
-### Description of the Change
-
-<!--
-
-We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts.
-
--->
-
-### Alternate Designs
-
-<!-- Explain what other alternates were considered and why the proposed version was selected -->
-
-### Why Should This Be In Core?
-
-<!-- Explain why this functionality should be in atom/atom as opposed to a package -->
-
-### Benefits
-
-<!-- What benefits will be realized by the code change? -->
-
-### Possible Drawbacks
-
-<!-- What are the possible side-effects or negative impacts of the code change? -->
-
-### Verification Process
-
-<!--
-
-What process did you follow to verify that your change has the desired effects?
-
-- How did you verify that all new functionality works as expected?
-- How did you verify that all changed functionality works as expected?
-- How did you verify that the change has not introduced any regressions?
-
-Describe the actions you performed (e.g., buttons you clicked, text you typed, commands you ran, etc.), and describe the results you observed.
-
--->
-
-### Applicable Issues
-
-<!-- Enter any applicable Issues here -->
+⚛👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.
+
+1. Copy the correct template for your contribution
+  - 🐛 Are you fixing a bug? Copy the template from https://bit.ly/atom-bugfix
+  - 📈 Are you improving performance? Copy the template from https://bit.ly/atom-perf
+  - 📝 Are you updating documentation? Copy the template from https://bit.ly/atom-docs
+  - 💻 Are you changing functionality? Copy the template from https://bit.ly/atom-behavior
```

Comment:
Note: These https://bit.ly links assume that the templates exist on Atom's master branch. Therefore, these links result in a 404 right now, but they'll work correctly once this PR is merged to master.